lib/new_relic/noticed_error.rb

```diff
@@ -27,6 +27,9 @@ class NewRelic::NoticedError
 
   DESTINATION = NewRelic::Agent::AttributeFilter::DST_ERROR_COLLECTOR
 
+  AGENT_ATTRIBUTE_REQUEST_URI = :'request.uri'
+  AGENT_ATTRIBUTE_ERROR_GRUP = :'error.group.name'
```

Comment:
lil typo "GRUP" => "GROUP"

Reply:
Thank you! Addressed with e4657bfb143a8199401740a0cd2259d28c79be23